src/main/java/com/esri/samples/featurelayers/feature_collection_layer_query/README.md

```diff
@@ -0,0 +1,38 @@
+<h1>Feature Collection Layer Query</h1>
+
+<p>Create a feature collection layer which shows the result of a SQL query from a service feature table.</p>
+
+<p><img src="FeatureCollectionLayerQuery.png"/></p>
+
+<h2>How to use the sample</h2>
+
+
+<p>Simply run the sample. The result of the SQL query from the hard coded service feature table URL (in this example, wildfire response point data hosted by ArcGIS) is then displayed on the map.</p>
+
+<h2>How it works</h2>
+
+<ol>
+    <li>Create a <code>ServiceFeatureTable</code> object from a URL, and store it as a <code>FeatureTable</code> object.</li>
+	<li>Create a <code>QueryParameters</code> object and specify the where clause with <code>QueryParameters.setWhereClause()</code> from the text entered by the user.</li>
```

Comment:
> text entered by the user

what text?

Reply:
this was to refer back to the condition entered by the user in the code, but I agree, it's confusing to refer to that here. I'll remove it.